apps/server/src/modules/learnroom/controller/course.controller.ts

```diff
@@ -1,13 +1,39 @@
 import { Authenticate, CurrentUser, ICurrentUser } from '@modules/authentication';
-import { Controller, Get, NotFoundException, Param, Query, Res, StreamableFile } from '@nestjs/common';
+import {
+	Controller,
+	FileTypeValidator,
+	Get,
+	MaxFileSizeValidator,
+	NotFoundException,
+	Param,
+	ParseFilePipe,
+	Post,
+	Query,
+	Res,
+	StreamableFile,
+	UploadedFile,
+	UseInterceptors,
+} from '@nestjs/common';
 import { ConfigService } from '@nestjs/config';
-import { ApiTags } from '@nestjs/swagger';
+import { FileInterceptor } from '@nestjs/platform-express';
+import {
+	ApiBadRequestResponse,
+	ApiBody,
+	ApiConsumes,
+	ApiCreatedResponse,
+	ApiInternalServerErrorResponse,
+	ApiOperation,
+	ApiTags,
+} from '@nestjs/swagger';
 import { PaginationParams } from '@shared/controller/';
+import { config } from '@src/modules/files-storage/files-storage.config';
```

Comment:
This config is for your area secret. You can only use parts that are exported by the module barrel and only if you import this module. In this case this is a stand alone module with one deployment.